build/targets.props

```
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <SrcTargets>net461;net462;net472;netstandard2.0;net6.0</SrcTargets>
+    <SrcTargets>net461;net462;net472;netstandard2.0;net6.0;net8.0</SrcTargets>
```

Maybe we should only enable the net8.0 target framework if people opt-in by setting an environment variable in their console or machine `USE_DOTNET_8_PREVIEW`

```suggestion
    <SrcTargets Condition="'$(USE_DOTNET_8_PREVIEW)' == 'true' ">net461;net462;net472;netstandard2.0;net6.0;net8.0</SrcTargets>
```

---

This is in its own issue now https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2111